docs/vitepress/guide/tutorial/html.md

```diff
@@ -13,7 +13,7 @@ We expose all Vuetify components in ***trame***. As an example, let's look at ho
 ```javascript
 // Somewhere in javascript
 const currentSuffix = "lbs";
-const myWeight = 28.0;
+const myWeight = 28.0;  // `let myWeight` or `ref(28.0)` ?
```

Comment:
@jourdain not to be merged, but I found it odd that `myWeight` was `const` here.

Reply:
yes that is odd, as they need to be vue/ref for them to be reactive.

Reply:
I meant...

Shouldn't `const myWeight = 28.0;` have been `let myWeight = 28.0;` or `const myWeight = 28.0;` ?

If `const myWeight = 28.0;` is fine for VueJS then I guess it's fine to leave it like this. 
In any case, no big deal.

Reply:
Just leave it like that, we don't want to teach the internals of vue.js

Reply:
if you want, you can use `let`

Reply:
either way, it is wrong and neither would truly work like that. But conceptually is what matters.

Reply:
gotcha... to be more accurate, we could have used a dictionary instead of standalone variables... but I guess that makes it slightly more confusing to understand the idea behind... let's forget about it.